### ISSUE TITLE
Android kiwi browser adn

### ISSUE THREAD

**Issue by OlivierDescoups** [polywock/globalSpeed#82]
Hello your extension did not work on animedigitalnetwork.fr 
I test it on android chrome extention browser : kiwi browser but the extension recognize the video and i can change the volume of the video

**Comment by polywock** [comment]
The speed isn't applying correctly? I just tried on that website and it works for me.

if the volume doesn't change with the volume hotkey but changes through popup controls, try prioritizing the video (arrow/cursor icon next to volume control). 

![](https://user-images.githubusercontent.com/31208859/115490756-ba224180-a22c-11eb-9fbf-9d6c2199ede0.png)

If it's a Android kiwi specific problem, I can't help out with that. I tried debugging with Android Kiwi before, but it's a bit of a nightmare.
